miro_preprocessor/xml_to_json_converter.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3-alpine3.6
```

Comment:
Minor: either `FROM alpine` and install python3 manually, or `FROM python:3-alpine3.6` and you can drop the explicit `apk add python3` – but not both.